src/ui/ApplicationSearch/SearchResultListItem.tsx

```diff
@@ -48,8 +58,15 @@ const SubText = styled.div`
   color: ${({ theme }) => theme.ui.searchResult.subText.color};
 `;
 
-const titleLink = (item: State.SearchItem, toggleModal?: () => void) => {
+const titleLink = (
+  item: State.SearchItem,
+  itStatus: any | undefined,
```

Comment:
A minor Typescript thing for you can mark an argument as a type or undefined as it being optional.. but then it has to be listed after any required arguments.. so this would look like;

```
const titleLink = (item: State.SearchItem, openDialog: () => void, itStatus?: any, toggleModal?: () => void)
```